homeassistant/components/islamic_prayer_times/strings.json

```diff
@@ -15,7 +15,7 @@
     "step": {
       "init": {
         "data": {
-          "calc_method": "Prayer calculation method"
+          "calculation_method": "Prayer calculation method"
```

Comment:
After merging this we should update the translation key in lokalise or else all existing translations will be lost .

Reply:
Is that done via the key editor?

https://docs.lokalise.com/en/articles/2059009-key-editor